index.html

```diff
@@ -11,6 +11,34 @@
     </script>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
+    <meta name="description" content="Nibble Computer Society">
+    <meta name="keywords" content="Nibble Computer Society is the official computer society of JSS Noida. We are a group of students working towards enhancing knowledge in the field of computer science. The website shows our contribution to the college, our members and events & workshops that we organize."/>
```

Comment:
These are not keywords. Keywords can be Nibble Computer Society, NCS, JSS Noida, JSS, student society, computer society, etc